.drone.yml

```diff
@@ -188,28 +241,39 @@ pipeline:
       - su-exec www-data ./run.sh --remote --type api --tags '~@skipOnEncryption&&~@skipOnEncryptionType:${ENCRYPTION_TYPE}&&~@skip'
```

Comment:
why not using `make` here?

Reply:
This is running core acceptance tests. The local ``make test-acceptance-*`` is designed for running local app acceptance tests. I didn't consider this a major item in this refactoring PR.

I will have a play with this as a separate issue, probably I can use the local ``make test-acceptance-*`` and pass it enough env vars to point to the core Behat config and suite.